webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/Table/Table.js

```diff
@@ -16,6 +16,7 @@ import Pagination from '../../../Pagination';
 import { DeleteModal } from './DeleteModal';
 import EmptyPage from '../../../../routes/common/EmptyPage';
 import { getColumnHelpers } from './helpers';
+import { RowSelectTd as DefaultRowSelectTd } from '../../../HostsIndex/RowSelectTd';
```

Comment:
Since its the default for all tables, can this move to the tableIndexPage folder?

Reply:
Done